app/views/drawings/_drawings.html.haml

```diff
@@ -1 +1,51 @@
-= render @drawings.desc
+.container
+  %h3 Manage Drawings
+  %table.table.table-bordered.table-condensed.table-hover.table-responsive
+    %thead
+      %tr
+        %th ID
+        %th File
+        %th Subject Matter
+        %th Description
+        %th Age
+        %th Artist's Gender
+        %th Story
+        %th Mood Rate
+        %th Edit
+        %th Delete
+        %th User Admin
+        %th Status
+    %tbody
+    - @drawings.each do |drawing|
+      %tr
+        %td
+          = drawing.id
+        %td
+          = image_tag drawing.image.url(:thumb)
+        %td
+          = drawing.subject_matter
+        %td
+          = drawing.description[0, 29].concat('...')
```

Comment:
Yep not necessary for now, but I should def have time to add it this weekend. Let's not let it block this PR merge in any case but I can create an issue if needed to helperize all the things. Especially as I actually would like to use decorators going forward via https://github.com/drapergem/draper for keeping logic out of the view (and it's a nicer way of adding presentation logic to model attributes instead of relying on helpers too much).